modules/cco-short-term-creds-auth-flow-aws.adoc

```diff
@@ -6,7 +6,49 @@
 [id="cco-short-term-creds-auth-flow-aws_{context}"]
 = AWS Security Token Service authentication process
 
-The following diagram details the authentication flow between AWS and the {product-title} cluster when using AWS STS.
+The AWS Security Token Service (STS) and the link:https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html[`AssumeRole`] API action allow pods to retrieve access keys that are defined by an IAM role policy.
+
+The {product-title} cluster includes a Kubernetes service account signing service. This service uses a private key to sign service account JSON web tokens (JWT). A pod that requires a service account token requests one through the pod specification. When the pod is created and assigned to a node, the node retrieves a signed service account from the service account signing service and mounts it onto the pod.
+
+Clusters that use STS contain an IAM role ID in their Kubernetes configuration secrets. Workloads assume the identity of this IAM role ID. The signed service account token issued to the workload aligns with the configuration in AWS, which allows AWS STS to grant access keys for the specified IAM role to the workload.
+
+AWS STS only grants access keys for requests that include service account tokens that meet the following conditions:
```

Comment:
In this sentence, are you trying to say that "granting access keys for requests that include service account tokens that meet the following conditions" is literally the only action that AWS STS does? Or is it meant to say that the AWS STS grants access keys, but only for requests that meet those conditions listed?

If the latter, I'd suggest moving "only" elsewhere:
```suggestion
AWS STS grants access keys only for requests that include service account tokens that meet the following conditions:
```

Reply:
aaaaah very astute and discerning, thank you!